test/xpu_api/iterators/predef.iterators/move.iterators/move.iter.ops/move.iter.op.compound.assign.minus/difference_type4.pass.cpp

```diff
@@ -0,0 +1,84 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
```

Comment:
difference_type4.pass.cpp => difference_type_minus.pass.cpp

Reply:
Renamed to another name.